test/test_valgrind.py

```diff
@@ -21,6 +21,8 @@ def check_valgrind(implementation: pqclean.Implementation):
     if (platform.machine() not in ('i386', 'x86_64') or
             platform.system() != 'Linux'):
         raise unittest.SkipTest()
+    if os.environ.get('CIRCLECI') == 'true' and os.environ.get('ARCH') == 'i386' and (implementation.scheme.name == 'frodokem1344aes' or implementation.scheme.name == 'frodokem1344shake'):
+        raise unittest.SkipTest('Skipping too-slow valgrind test')
```

Comment:
Not sure if we want this all the time. For SPHINX it will be even worse - if valgrind proves too slow, we might want to just remove it from all CI tests.

Also, only disable it on CircleCI? What about Appveyor/Travis?

Reply:
Yes we need to decide which tests we're running on which CI environments. But let's save that for an issue on its own.

Reply:
But don't you intend to also not run the valgrind test on Travis and Appveyor? Shouldn't you do `'CI' in os.environ`?

Reply:
It wasn't running too slow on Travis and Appveyor -- it's only the qemu i386 CircleCI builds that it was running too slow on.